FirebaseMessaging/Apps/SampleStandaloneWatchApp/SampleStandaloneWatchAppWatchApp/SampleStandaloneWatchAppApp.swift

```diff
@@ -0,0 +1,55 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import SwiftUI
+import Firebase
+
+@main
+struct SampleStandaloneWatchApp_Watch_AppApp: App {
+  @WKApplicationDelegateAdaptor(FCMWatchAppDelegate.self) var appDelegate
+  var body: some Scene {
+    WindowGroup {
+      ContentView()
+    }
+  }
+}
+
+// MARK: AppDelegate
+
+class FCMWatchAppDelegate: NSObject, WKApplicationDelegate, MessagingDelegate {
+  func applicationDidFinishLaunching() {
+    FirebaseApp.configure()
+    let notifCenter = UNUserNotificationCenter.current()
+    notifCenter.requestAuthorization(options: [.alert, .sound]) { granted, error in
+      if granted {
+        WKApplication.shared().registerForRemoteNotifications()
+      }
+    }
+    Messaging.messaging().delegate = self
+  }
+
+  func didRegisterForRemoteNotifications(withDeviceToken deviceToken: Data) {
+    print("APNS didRegisterForRemoteNotifications. Got device token \(deviceToken)")
```

Comment:
Nit: cant we add a comment here:
    /// Swizzling should be disabled in Messaging for watchOS, set APNS token manually.

Reply:
+1 to that. Let's also please link to the instructions for disabling Swizzling, or alternatively write them down in the comment.

Reply:
Done - added the comment and a link to our documentation page.